core/data/src/main/java/com/yapp/core/data/remote/api/OperationsApi.kt

```diff
@@ -16,4 +18,10 @@ internal interface OperationsApi {
 
     @GET("v1/operations/links/privacy-policy")
     suspend fun getPrivacyPolicyLink(): LinkResponse
+
+    @GET("/v1/operations/force-update")
+    suspend fun isForceUpdateRequired(
+        @Query("version") version: String,
+        @Query("platform") platform: String = "Android"
+    ): ForceUpdateResponse
```

Comment:
C: 슬래시가 중복으로 들어갔어요 ~

```suggestion
    @GET("v1/operations/force-update")
    suspend fun isForceUpdateRequired(
        @Query("version") version: String,
        @Query("platform") platform: String = "Android"
    ): ForceUpdateResponse
```

Reply:
아 ... 무지성 스웨거 복붙하다보니 .. ㅎㅎ 매의 눈이시군요 확인 감사합니다